showcase/src/app/demo-ui/pages/generic-search/demo-generic-search-page.component.html

```diff
@@ -0,0 +1,35 @@
+<h1 class="mat-display-3" translate>SHOWCASE.DEMO.GENERIC_SEARCH.TITLE</h1>
+<section class="stark-section">
+	<h1 translate>SHOWCASE.DEMO.SHARED.EXAMPLE_VIEWER_LIST</h1>
+	<example-viewer
+		[extensions]="['ts', 'html']"
+		filesPath="generic-search/demo-generic-search-page.component"
+		[exampleTitle]="'SHOWCASE.DEMO.GENERIC_SEARCH.BASIC_IMPLEMENTATION' | translate"
+	>
+		<mat-slide-toggle color="primary" [checked]="!hideSearch" (change)="hideSearch = !hideSearch">
+			{{ "SHOWCASE.DEMO.GENERIC_SEARCH.TOGGLE" | translate }}
+		</mat-slide-toggle>
+		<stark-generic-search
+			formHtmlId="demo-generic-search-form"
+			(searchTriggered)="onSearch($event)"
+			(resetTriggered)="onReset($event)"
+			[isFormHidden]="hideSearch"
+		>
+			<demo-generic-search-form
+				#searchForm
+				[searchCriteria]="workingCopy"
+				(workingCopyChanged)="updateWorkingCopy($event)"
+			></demo-generic-search-form>
+		</stark-generic-search>
+		<stark-table
+			title="SHOWCASE.DEMO.GENERIC_SEARCH.SEARCH_RESULTS"
```

Comment:
This is in fact the tooltip of the Table.... I was talking about a title for the table like we do in the demo page of the Table component

Reply:
Ho, my bad 😞
It's definitely a complex PR...